Update Configure MySQL for 4-byte Unicode Support (#841)
owncloud/docs-server: modules/admin_manual/pages/configuration/database/linux_database_configuration.adoc

==== Configure MySQL for 4-byte Unicode Support

For supporting such features as emoji, both MySQL (or MariaDB) *and* ownCloud need to be

> capable using, respectively if necessary, configured to use 4-byte Unicode support instead of the default 3-byte.

Reviewer: ```suggestion
capable of using 4-byte Unicode instead of the default 3-byte and configured accordingly.
```
